### PR TITLE
Obisindicators tutorial

### PR DESCRIPTION
**Pull request by Marie59**
Initiating a PR for the tutorial on conducting a study on marine data from obis to calculate ocean biodiversity indicators.

### CODE REVIEW

File: topics/ecology/tutorials/obisindicators/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+## Upload of the data
+
+This first step consist of downloading and uploading obis data onto galaxy.
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial and give it a name (example: “Obisindicators tutorial”) for you to find it again later if needed.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md box_type="none" %}
```
Comment (hexylena) [inline topics/ecology/tutorials/obisindicators/tutorial.md:50]:
```suggestion
>    {% snippet faqs/galaxy/histories_create_new.md %}
```

this is intended to be a tip box (so experienced galaxy users can skip over it. Is there a reason to remove the box type here?

Comment (Marie59) [reply topics/ecology/tutorials/obisindicators/tutorial.md:50]: No reason at all ! I think it's an unfortunate copy paste ;)

**Review by yvanlebras — DISMISSED** [review]

**Comment by yvanlebras** [comment]
Is it possible to look if it is ok @shiltemann ?

File: topics/ecology/tutorials/obisindicators/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+title: Obis marine indicators
+zenodo_link: ''
+questions:
+objectives:
+- Calculating and vizualizing marine biodiversity indicators
+time_estimation: 1H
+key_points:
+- Marine data
+- Diversity indicators
+contributors:
```
Comment (hexylena) [inline topics/ecology/tutorials/obisindicators/tutorial.md:13]: If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Comment (hexylena) [reply topics/ecology/tutorials/obisindicators/tutorial.md:13]: (8f this is for gallantries please indicate accordingly)

**Comment by Marie59** [comment]
I don't really understand the remaining issues. Is it coming from the main_workflow.ga ? @yvanlebras an idea ?

**Comment by hexylena** [comment]
@Marie59 yes, it comes from the workflow. If you edit the workflow in Galaxy, you'll see a menu for "best practices" which lets you add important metadata like a license, in addition to creator information (i.e. you). 

Additionally the workflow is missing a test case, we have started trying to require those for new tutorials, as they help us ensure these tutorials work across usegalaxy.*. If you've run this workflow once in your history, then generating the tests is very easy: https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html

**Comment by Marie59** [comment]
Great ! Thank you I'm on it for both here and remote sensing then !

**Comment by Marie59** [comment]
@hexylena did I do everything right ? Thanks a lot for your help !

**Comment by hexylena** [comment]
Yes looks good :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**